h/httpserver.h

```diff
@@ -212,6 +212,7 @@ typedef struct HttpServer_tag{
   char             *defaultProductURLPrefix;
   uint64           serverInstanceUID;   /* may be something smart at some point. Now just startup STCK */
   void             *sharedServiceMem; /* address shared by all HttpServices */
+  hashtable        *PluginLoggingTable;
```

Comment:
* A variable name should not start with a capital letter
* This doesn't belong in the http server. A plug-in is a ZSS feature, not a feature of the http server. You can make it a dynamic property in the http server. So don't need to make any changes in httpserver.c/h. See ```privilegedServerName``` in https://github.com/zowe/zss as a reference.

Reply:
As of now, the data service would take that hashtable and use it to get the logging component id associated with the plugin identifier. This can be moved to a dynamic property without an problems.